### PR TITLE
RDM-9995-Create service-bus.tf

### PR DESCRIPTION
**Pull request by nathan-clark**
Create Azure Service Bus Namespace/Topic, for work allocation

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: service-bus.tf [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+module "servicebus-namespace" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-namespace?ref=master"
+  name                = local.servicebus_namespace_name
+  location            = var.location
+  resource_group_name = azurerm_resource_group.rg.name
+  env                 = var.env
+  common_tags         = local.tags
+  sku                 = "Premium"
```
Comment (mario-paniccia) [inline service-bus.tf:14]: when we enable Premium do we have by default High Availability? or we need to explicitly configure for HA?

**Review by mario-paniccia — DISMISSED** [review]
👍

**Review by mario-paniccia — COMMENTED** [review]
unapproving this since we need to add configuration for High Availability

File: service-bus.tf (hunk 1 above)
Comment (nathan-clark) [reply service-bus.tf:14]: `  zoneRedundant       = true` now @mario-paniccia

Comment (mario-paniccia) [reply service-bus.tf:14]: Could we make this configurable per environment? I guess on Demo for example we won't need `zoneRedundant`?

Comment (nathan-clark) [reply service-bus.tf:14]: we can remove this var in the specific inf branches

File: service-bus.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+locals {
+  events_topic_name         = "${var.product}-events-topic-${var.env}"
```
Comment (mario-paniccia) [inline service-bus.tf:2]:
```suggestion
  events_topic_name         = "${var.product}-case-events-topic-${var.env}"
```

**Pull request closed without merging**